scriptmodules/supplementary/esthemes.sh

```diff
@@ -51,7 +51,8 @@ function gui_esthemes() {
         'ehettervik pixel-metadata'
         'ehettervik pixel-tft'
         'ehettervik luminous'
-        'ehettervik minilumi'
+
+'ehettervik minilumi'
```

Comment:
you have broken the indentation. I will add your theme manually when I have a change.